src/Arcus.WebApi.Security/Authorization/JwtTokenAuthorizationOptions.cs

```diff
@@ -36,6 +37,15 @@ public JwtTokenAuthorizationOptions()
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="JwtTokenAuthorizationOptions"/> class.
+        /// </summary>
+        /// <param name="claimCheck">Custom claims key-value pair to validate against.</param>
+        public JwtTokenAuthorizationOptions(IDictionary<string, string> claimCheck = null)
```

Comment:
We may consider an constructor overload instead of an optional parameter because it will result in `MethodNotFound` exceptions when the packages are updated seperatly.

Reply:
done